pymdownx/blocks/block.py

```diff
@@ -1,9 +1,17 @@
 """Block class."""
+from __future__ import annotations
 from abc import ABCMeta, abstractmethod
 import functools
 import copy
 import re
+import sys
 from markdown import util as mutil
+import xml.etree.ElementTree as etree
+from typing import Any, Callable, TypeVar, TYPE_CHECKING
+from collections.abc import Iterable
+
+if TYPE_CHECKING:
```

Comment:
Please add the comment `# pragma: no cover` on the if line to ignore code coverage here.